connect/transforms/src/main/java/org/apache/kafka/connect/transforms/ReplaceField.java

```diff
@@ -72,6 +73,10 @@ interface ConfigName {
             .define(ConfigName.REPLACE_NULL_WITH_DEFAULT_CONFIG, ConfigDef.Type.BOOLEAN, true, ConfigDef.Importance.MEDIUM,
                     "Whether to replace fields that have a default value and that are null to the default value. When set to true, the default value is used, otherwise null is used.");
 
+    private static Set<String> configNames() {
```

Comment:
I note that `ConfigName.RENAME` actually has the value `"renames"` so it probably should be `ConfigName.RENAMES`. This probably ought to be resolved.

Reply:
@TaiJuWu Maybe you can file a minor PR to fix the naming. this PR needs more discussion due to behavior change.

Reply:
> @TaiJuWu Maybe you can file a minor PR to fix the naming. this PR needs more discussion due to behavior change.

Will do, thanks for helping check.